e2e/pages/enterChildren/enterChildren.js

```diff
@@ -45,43 +46,45 @@ module.exports = {
     activeChild = 'additionalChildren_0';
   },
 
-  enterChildDetails(name, day, month, year, gender = 'Boy') {
-    I.fillField(this.fields(activeChild).fullName, name);
-    I.fillField(this.fields(activeChild).DOB.day, day);
-    I.fillField(this.fields(activeChild).DOB.month, month);
-    I.fillField(this.fields(activeChild).DOB.year, year);
-    I.selectOption(this.fields(activeChild).gender, gender);
+  enterChildDetails(child) {
+    I.fillField(this.fields(activeChild).fullName, child.name);
+    I.fillField(this.fields(activeChild).DOB.day, child['DOB'].day);
+    I.fillField(this.fields(activeChild).DOB.month, child['DOB'].month);
+    I.fillField(this.fields(activeChild).DOB.year, child['DOB'].year);
+    I.selectOption(this.fields(activeChild).gender, child.gender);
   },
 
-  defineChildSituation(day, month, year, situation = 'Living with respondents', address = '35 London Lane') {
-    I.selectOption(this.fields(activeChild).situation.selector, situation);
-    I.fillField(this.fields(activeChild).situation.dateStartedStaying.day, day);
-    I.fillField(this.fields(activeChild).situation.dateStartedStaying.month, month);
-    I.fillField(this.fields(activeChild).situation.dateStartedStaying.year, year);
-    I.fillField(this.fields(activeChild).situation.addressOfChild, address);
+  defineChildSituation(child) {
+    I.selectOption(this.fields(activeChild).situation.selector, child.situation);
+    I.fillField(this.fields(activeChild).situation.dateStartedStaying.day, child['DOB'].day);
+    I.fillField(this.fields(activeChild).situation.dateStartedStaying.month, child['DOB'].month);
+    I.fillField(this.fields(activeChild).situation.dateStartedStaying.year, child['DOB'].year);
+    within(this.fields(activeChild).situation.addressOfChild, () => {
+      postcodeLookup.lookupPostcode(child);
+    });
   },
 
-  enterKeyDatesAffectingHearing(keyDates = 'Tuesday the 11th') {
-    I.fillField(this.fields(activeChild).keyDates, keyDates);
+  enterKeyDatesAffectingHearing(child) {
```

Comment:
Please revert. Methods should accept only what is absolutely needed, nothing more.